persist-cli/src/main/java/io/ballerina/persist/nodegenerator/BalFileConstants.java

```diff
@@ -232,6 +232,11 @@ private InheritedTypeReferenceConstants() {
             " <%sRelations[]>self.include);";
     public static final String PLACEHOLDER_PATTERN = "\"\\$\\{(.*?)}\"";
     public static final String MYSQL_DRIVER_IMPORT = "import ballerinax/mysql.driver as _;";
+
+    public static final String AUTOGENERATED_FILE_COMMENT = "// AUTO-GENERATED FILE. DO NOT MODIFY.\n" +
+            "\n" +
+            "// This file is a auto-generated client by Ballerina persist library.\n" +
```

Comment:
Can this blanket statement be used in all files? database_configuration.bal file does not contain clients.

```suggestion
            "// This file is a auto-generated by Ballerina persistence layer for {0}.\n" +
```

Reply:
Updated in [4daf0c6](https://github.com/ballerina-platform/persist-tools/pull/79/commits/4daf0c6807a98caf258c2ef37d22f7361f78dad1)